keystorage/azure/src/main/java/tech/pegasys/signers/azure/AzureKeyVault.java

```diff
@@ -49,4 +64,18 @@ public AzureKeyVault(
       return Optional.empty();
     }
   }
+
+  public CryptographyClient fetchKey(final String keyName, final String keyVersion) {
```

Comment:
I think you can make `keyVersion` optional. If not specified, it should return latest version via `keyClient.getKey(keyName)`

Reply:
The azure API states that if the version is null, or empty, defaults to latest - is that good enough?

Reply:
👍  ... assuming keyVersion can be omitted in config.